### PR TITLE
Add missing DOMCharacterData::insertData() return false description

### PR DESCRIPTION
**Pull request by othercorey**

### CODE REVIEW

**Comment by othercorey** [comment]
@cmb69 I will just close if these aren't acceptable. I assume you wanted more info. I was just dropping missing return descriptions.

File: reference/dom/domcharacterdata/insertdata.xml [hunk 1]
```diff
@@ -44,7 +44,7 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   &return.void;
+   Returns &false; on error.
```
Comment (cmb69) [inline reference/dom/domcharacterdata/insertdata.xml:47]: And `true` on success, I think, in which case we should have:
```suggestion
   &return.success;
```

**Review by cmb69 — COMMENTED** [review]
Thank you!

File: reference/dom/domcharacterdata/insertdata.xml (hunk 1 above)
Comment (othercorey) [reply reference/dom/domcharacterdata/insertdata.xml]: Updated all 3 DOMCharacterData PRs with this change.

**Comment by othercorey** [comment]
@cmb69 I think these are all ready to go.

**Pull request merged**